pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/apps.kruise.io/v1alpha1/AdvancedCronJob/customizations_tests.yaml

```diff
@@ -0,0 +1,18 @@
+tests:
+  - desiredInputPath: testdata/desired-acj-nginx.yaml
+    statusInputPath: testdata/status-file.yaml
+    operation: AggregateStatus
+  - desiredInputPath: testdata/desired-acj-nginx.yaml
+    operation: InterpretDependency
+#  - observedInputPath: testdata/observed-acj-nginx.yaml
```

Comment:
What do you think we need to comment instead of deletion when some operations do not need to be verified?